string tag_frame_id
---
geometry_msgs/Pose pose
bool status
